geomesa-blobstore/geomesa-blobstore-core/src/main/java/org/locationtech/geomesa/blob/core/interop/GeoMesaBlobStore.java

```diff
@@ -0,0 +1,72 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+
+package org.locationtech.geomesa.blob.core.interop;
+
+import org.geotools.data.Query;
+import org.opengis.filter.Filter;
+import scala.Option;
```

Comment:
yeah. actually isn't the convention in Java usually to return a null and doc it in the javadoc ? Yeah. follow this convention: https://docs.oracle.com/javase/7/docs/api/java/util/Map.html#put(K,%20V) perhaps